src/assets/translations/fr/main.json

```
@@ -668,7 +668,8 @@
       "noRateAvailable": "Cela est souvent dû à une liquidité très limitée ou inexistante sur la paire d'échange.",
       "minerFee": "Il s'agit des frais de traitement de la transaction sur la blockchain, ce ne sont pas des frais facturés par ShapeShift.",
       "donation": "Aidez à soutenir ShapeShift et maintenez-nous opérationnels.",
-      "protocol": "C'est le protocole avec lequel votre échange est acheminé.",
+      "protocol": "C'est le protocole avec lequel votre échange est routé.",
+      "protocolLifi": "C'est le protocole par lequel votre échange est routé. LI.FI exécute une série de transactions intermédiaires sous le capot pour la meilleure combinaison globale de prix, de sécurité et de rapidité.",
```

Perhaps a better wording, 1/1 translation from the original one makes it weird IMO ("sous le capot")
```suggestion
      "protocolLifi": "C'est le protocole par lequel votre échange est routé. LI.FI exécute une série de transactions intermédiaires pour trouver la meilleure combinaison globale de prix, de sécurité et de rapidité.",
```

---

OK committed, [but it is an idiom](https://fr.wiktionary.org/wiki/sous_le_capot) in French too ;)